kolibri/core/package.json

```diff
@@ -38,5 +38,10 @@
     "vuex": "^3.6.2",
     "vuex-router-sync": "^5.0.0",
     "xstate": "^4.38.3"
+  },
+  "devDependencies": {
```

Comment:
These should be added as dependencies to the kolibri-tools package, rather than being added here.

Reply:
Got it. Will update. Thanks :)